### PR TITLE
Run lint on commonserverpython

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25125

## Must have
- [x] Tests

### CODE REVIEW

File: demisto_sdk/commands/lint/tests/test_linter/gather_facts_test.py [hunk 1]
```diff
@@ -38,6 +38,33 @@ def test_not_valid_yaml(self, demisto_content: Callable, create_integration: Cal
...
+        - Repo with CommonServerPython script
+
+        When
+        - Running lint on a repo with a change to CommonServerPython
+
+        Then
+        - Validate that the CommonServerPython is in the file list to check
+        """
+        from demisto_sdk.commands.lint import linter
+        from wcmatch.pathlib import Path
```
Comment (Itay4) [inline demisto_sdk/commands/lint/tests/test_linter/gather_facts_test.py:54]: is there a reason this is imported in every test?

Comment (jochman) [reply demisto_sdk/commands/lint/tests/test_linter/gather_facts_test.py:54]: That's how it worked before. I can move it up and change this file.

**Comment by Itay4** [comment]
not sure when we stopped running flake8 on commonserver, need to verify it passes so we wont break

File: demisto_sdk/commands/lint/tests/test_linter/gather_facts_test.py (hunk 1 above)
Comment (Itay4) [reply demisto_sdk/commands/lint/tests/test_linter/gather_facts_test.py:54]: lets move it, it doesnt make sense

**Comment by jochman** [comment]
> not sure when we stopped running flake8 on commonserver, need to verify it passes so we wont break

It will break as commonmsp is full of errors:
image

**Comment by Itay4** [comment]
thats why im saying lets fix content before merging this

**Pull request merged**